include/envoy/upstream/outlier_detection.h

```diff
@@ -129,9 +130,9 @@ enum class EjectionType { Consecutive5xx, SuccessRate, ConsecutiveGatewayFailure
 /**
  * Sink for outlier detection event logs.
  */
-class EventLogger {
+class OutlierDetectionEventLogger {
```

Comment:
You are already in namespace 'Outlier' so maybe it's sufficient to call this DetectionEventLogger ?

Reply:
@jmarantz @junr03 initially took similar approach as in `HealthCheckEventLogger`. `DetectionEventLogger` in my opinion is better